hack/ci-e2e.sh

```diff
@@ -79,6 +81,12 @@ while ! minikube start; do sleep 30; done
 # Temporary workaround for https://github.com/kubernetes/minikube/issues/18021
 docker image save -o /tmp/bmo-e2e.tar quay.io/metal3-io/baremetal-operator:e2e
 minikube image load /tmp/bmo-e2e.tar
+if [[ "${LOCAL_IRONIC}" == "true" ]]; then
+echo "Saving local ironic image!"
+docker image save -o \
```

Comment:
What happens if this image doesn't exist? Shouldn't we build it here first?

Reply:
IMO we shouldn't integrate external image building into the BMO test code. This option won't be used in CI rather in custom tests done by a developer manually. Whoever is doing the test has the responsibility to build an image locally .

In other words IMO building an Ironic image and managing build arguments etc for Ironic is out of the scope of the BMO test code.

Reply:
I agree. We could perhaps rename the variable to "LOAD_IRONIC_IMAGE" or something like that to make it clear that we are just loading it. Not a big deal to me though. I think this is pretty self explanatory and not something people would randomly stumble on.

Reply:
Thanks for the reviews, I have changed it to LOAD_IRONIC_IMAGE.